config/F2.config

```diff
@@ -6,19 +6,19 @@
 process {
     withName: run_validate_PipeVal {
         cpus = 1
-        memory = 1.GB
+        memory = 2.GB
```

Comment:
For these processes that require 1 cpu, may as well give half of the memory.  Two of these processes can run at a time and no others.

Reply:
The F-series nodes don't have exactly 2xCPUs memory available so this will likely result in only one process of PipeVal running at a time

Reply:
Ah right!   I changed them to 1500.MB